cypress/fixtures/protocolData.json

```diff
@@ -0,0 +1,31 @@
+{
+  "matic": {
+    "DepositSize": "4 hour flowrate",
+    "OwedDepositSize": "4 hour flowrate",
+    "PatricianPeriod": "30 minutes",
+    "MinimumExitPeriod": "1 week",
+    "DefaultExitPeriod": "4 weeks",
+    "Resolver": "0xE0cc76334405EE8b39213E620587d815967af39C",
+    "CFAv1": "0x6EeE6060f715257b970700bc2656De21dEdF074C",
+    "SuperTokenFactory": "0x2C90719f25B10Fc5646c82DA3240C76Fa5BcCF34",
+    "TOGA": "0x6AEAeE5Fd4D05A741723D752D30EE4D72690A8f7",
+    "Host": "0x3E14dC1b13c488a8d5D310918780c983bD5982E7",
+    "IDAv1": "0xB0aABBA4B2783A72C52956CDEF62d438ecA2d7a1",
+    "SuperfluidLoaderV1": "0x15F0Ca26781C3852f8166eD2ebce5D18265cceb7"
+  },
+
+  "optimism-mainnet": {
```

Comment:
Lets use a testnet instead of 2 mainnets for this case since the periods are shown depending on that and since this is hardcoded we could grab it from the console code instead of making a fixture